tutorials/time-freq/plot_ssvep.py

```diff
@@ -0,0 +1,780 @@
+"""
+=======================================
+Basic analysis of an SSVEP/vSSR dataset
+=======================================
+
+In this tutorial we compute the frequency spectrum and quantify signal-to-noise
+ratio (SNR) at a target frequency in EEG data recorded during fast periodic
+visual stimulation (FPVS).
+Extracting SNR at stimulation frequency is a simple way to quantify frequency
+tagged responses in MEEG (a.k.a. steady state visually evoked potentials,
+SSVEP, or visual steady-state responses, vSSR in the visual domain,
+or auditory steady-state responses, aSSR in the auditory domain).
```

Comment:
The a is capitalised in ASSRs.
```suggestion
or auditory steady-state responses, ASSR in the auditory domain).
```

Reply:
> This is awesome. And I think its great that although you are doing VSSEPs you included terms such as ASSR, this should improve search engine hits.
> 
> I think their might be some differences in terminology between the visual and auditory fields in the topic of steady state responses. I had never heard of frequency tagging, so I already learnt a lot from this tutorial. Another example is that in ASSRs we typically use[ F-tests and Hotelling T^2 tests](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3387309/), whereas you seem to use a t-test. Once the tutorial is building in the CI do you mind if I suggest some additional terms in the text that might help ASSR researchers find this great tutorial?

thanks for the nice feedback and the suggestions @rob-luke ! 
happy to include the additional stats as soon as the the tutorial builds successfully.

agreed, the range of terminology with this methodology seem alomost unlimited :) 
it's one of the goals of the tutorial to include as many of these keywords as possible and keep it general purpose.
in terms of basic analysis it doesnt really matter which sensory domain you used to elicit a steady-state/frequency-tagged response